exercises/linked-list/linked_list.py

```diff
@@ -1,3 +1,4 @@
+# Represents smallest unit of a list with value, reference to succeding Node and referenece previous Node
```

Comment:
This might be more clear as `# A Node stores a value in the doubly linked list, as well as a link to any previous and and succeeding Nodes`.